manim/mobject/geometry/shape_matchers.py

```diff
@@ -104,21 +104,19 @@ def pointwise_become_partial(self, mobject, a, b):
         self.set_fill(opacity=b * self.original_fill_opacity)
         return self
 
-    def set_style(
-        self,
-        stroke_color=None,
-        stroke_width=None,
-        fill_color=None,
-        fill_opacity=None,
-        family=True,
-    ):
+    def set_style(self, fill_opacity, **kwargs):
         # Unchangeable style, except for fill_opacity
+        # All other style arguments are ignored
         super().set_style(
             stroke_color=BLACK,
             stroke_width=0,
             fill_color=BLACK,
             fill_opacity=fill_opacity,
         )
+        if len(kwargs) > 0:
+            logger.info(
+                "set_style for BackgroundRectangle does not process arguments other than fill_opacity."
```

Comment:
```suggestion
                "Arguments %s are ignored in BackgroundRectangle.set_style.",
                kwargs
```

Just slightly more verbose output.